### DISCUSSION TITLE
SSL Certificate problem on custom domain

### DISCUSSION THREAD

**Discussion by crisvini** [vercel/community#3637] [Help] [answered]
### Summary

I'm hosting a react project on Vercel using my custom domain from Google Domains but I'm havind some problem with SSL Certificate that I don't know how to solve. It's been a few days since I configurated my custom domain into Vercel but I still have the same problem and I don't know why becaus it seens to be everything right.

This is the error: NET::ERR_CERT_COMMON_NAME_INVALID

Here's my project domain settings at Vercel:
![image](https://github.com/vercel/community/assets/50123796/27c0e2c2-8fd8-4f7c-aed1-fb7e74add4fa)

### Example

https://cristianleoncini.dev/

### Steps to Reproduce

I'm just trying to use a custom domain in a react project, I configurated this using Vercel documentation and I don't get any error from Vercel.

**Comment by amyegan** [comment, marked as answer]
Hey @crisvini. It looks that domain has an `_acme-challenge` record. This record allows Let's Encrypt to resolve somewhere else, and it seems to be what is blocking certificate generation. The certificate should generate after you remove that DNS record 🙂

**Reply by crisvini** [reply]
thanks!
